app/routers/product.py

```diff
@@ -24,6 +24,12 @@ def add_new_product(payload: schema.Product, db: Session = Depends(get_db)):
     return {"data": new_product}
 
 
+@router.get("/")
```

Comment:
Can you explain this method

Reply:
this method provide information when requested using GET.